server/storage/wal/repair.go

```diff
@@ -77,7 +77,7 @@ func Repair(lg *zap.Logger, dirpath string) bool {
 			}
 
 			if _, err = io.Copy(bf, f); err != nil {
-				lg.Warn("failed to copy", zap.String("from", f.Name()+".broken"), zap.String("to", f.Name()), zap.Error(err))
+				lg.Warn("failed to copy", zap.String("from", f.Name()), zap.String("to", f.Name()+".broken"), zap.Error(err))
```

Comment:
Thank you for catching. 

Could you, please, use the temporary variable in all the 3 places:

```
 toName := f.Name() + ".broken"
```

Reply:
Thank you.